src/widgets/plots/pie_plot.py

```diff
@@ -11,8 +11,13 @@ def plot(self):
             ax = self.canvas.figure.subplots()
             counts = self.data.value_counts().to_dict()
             data_size = self.data.size
-            labels = [k if counts[k]/data_size > self.min_pie_plot_label_ratio else '' for k in counts.keys()]
-            ax.pie(counts.values(), labels=labels)
+            labels = [k for k in counts.keys() if counts[k]/data_size > self.min_pie_plot_label_ratio]
+            values = [counts[label] for label in labels]
+            values_sum = sum(values)
+            if values_sum < data_size:
+                labels.append('Other')
+                values.append(data_size-values_sum)
+            ax.pie(values, labels=labels)
```

Comment:
It is possible to get one blue circle with label 'Other'. I think you must add at least one label before add 'Other'.
![image](https://user-images.githubusercontent.com/72767517/169372520-a7a7bc07-5957-46c0-9af4-0a1a38d53f5b.png)